lib/autoprefixer.js

```diff
@@ -28,40 +28,52 @@ function timeCapsule(result, prefixes) {
     );
 }
 
-module.exports = postcss.plugin('autoprefixer', (...reqs) => {
+function getOptions(...args) {
```

Comment:
Why not just `getOptions(args)`? Faster and simpler ;).

Reply:
Exactly!